deepchem/models/torch_models/scscore.py

```diff
@@ -0,0 +1,48 @@
+import torch
```

Comment:
Could you add unit tests for these classes?

Reply:
Done !